test/lib/users/ssh_authorized_keyring_test.rb

```diff
@@ -90,122 +130,163 @@ def authorized_keys_from_home(path)
   describe "#write_keys" do
     let(:tmpdir) { Dir.mktmpdir }
     let(:home) { File.join(tmpdir, "/home/user") }
-    let(:file) { double("file", save: true) }
+    let(:file) { Yast::Users::SSHAuthorizedKeysFile.new(authorized_keys_path) }
     let(:ssh_dir) { File.join(home, ".ssh") }
     let(:key) { "ssh-rsa 123ABC" }
     let(:authorized_keys_path) { File.join(home, ".ssh", "authorized_keys") }
 
-    before { FileUtils.mkdir_p(home) }
+    before do
+      FileUtils.mkdir_p(ssh_dir)
+
+      allow(Yast::Users::SSHAuthorizedKeysFile).to receive(:new).and_return(file)
```

Comment:
this looks strange. Code reads real path and here content is ignored. Why not write content instead? So it tests on real file content?